### PR TITLE
feat(build): build both external and internal versions

### PR DESCRIPTION
**Pull request by JamyDev**

### CODE REVIEW

File: build/docs.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ const _ = require('lodash');
 const Bluebird = require('bluebird');
 const del = require('del');
 
+const argv = require('yargs').argv;
```
Comment (SimonSchick) [inline build/docs.js]: Do we really need to add a whole dependency for just a single startup param? I'd actually rather have a string includes check, unless we plan on refactoring the current system to fully use yargs.

**Comment by SimonSchick** [comment]
@JamyDev I left a comment, I'd like you to at least respond to it.

**Comment by JamyDev** [comment]
Oops, didn't see that. It's a build script, I don't think dependencies are such a problem. Especially if it makes the code nicer to read than `process.argv.includes('--includes')`. Just my 0.02.

**Comment by SimonSchick** [comment]
Doesn't really address the issue, dependency bloat is a problem and I'd like to avoid it when we can.

**Comment by ProbablePrime** [comment]
Seeing as this entire repo will be changing at some point this year into a completely new system that doesn't rely on Gulp. I'm not fussed by the tiny bloat of yargs which is most likely already on the dependency tree somewhere.

**Comment by JamyDev** [comment]
yargs is already included by gulp-less and gulp-uglify (via uglify-js). So technically I didn't add any dependency 😀

**Pull request merged**